lib/dialects/abstract/query-generator.js

```diff
@@ -414,7 +417,8 @@ var QueryGenerator = {
       query += ' LIMIT ' + this.escape(options.limit) + ' ';
     }
 
-    if (this._dialect.supports.returnValues) {
+    // Never return updated values in Oracle
+    if (this._dialect.supports.returnValues && this._dialect.name !== 'oracle') {
```

Comment:
You should use dialect supports mechanism like https://github.com/sequelize/sequelize/blob/54556cc4f80513b34fb5ab6a5fd0866387f6c706/lib/dialects/postgres/index.js#L31